QA.md

```diff
@@ -140,22 +169,22 @@ You can connect your wallet to the testnet (https://node1.foxtrot.testnet.hathor
 1. **Register tokens with same name**
     1. Unregister TST token but copy its configuration string and save it.
     1. Click on 'Custom tokens' and then 'Create a new token'.
-    1. Type 'Test token' in the token name, 'Test' in the symbol and 200 in the amount.
+    1. Type 'Test token' in the token name, 'TST' in the symbol and 200 in the amount.
```

Comment:
For this step, the symbol don't necessarily needs to be the same as the previous one as the check is on the token name

I'm not agains't this change tho

Reply:
If only the name needs to be the same, I would rather revert my change to keep this clear. :+1: